modules/virt-building-vm-containerdisk-image.adoc

```diff
@@ -129,7 +163,7 @@ EOF
 +
 [source,terminal]
 ----
-$ virt-customize -a <UUID>.qcow2 --run=customize-vm --firstboot=first-boot --selinux-relabel
```

Comment:
If `-firstboot=first-boot` is removed, then the section that creates the first boot script, should be removed as well.

Reply:
good catch! DONE